examples/data-sources/maas_boot_source/data-source.tf

```
@@ -0,0 +1,3 @@
+data "maas_boot_source" "default" {
```

Since the URL is read-only

```suggestion
data "maas_boot_source" "default" {}
```

---

And you also need to re-run the creation of the documentation